README.md

```diff
@@ -73,20 +99,60 @@ OpenCensus::Trace.in_span "my_task" do |span|
 end
 ```
 
+See the documentation for the `OpenCensus::Trace` module for more info.
```

Comment:
Link to API docs. http://opencensus.io/opencensus-ruby/api/OpenCensus/Trace.html

Reply:
Done